### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by mp1pro** [bessiewng/Jammming#3]
Noticed a lot of original and intuitive coding. The approach was cautious as I noticed a lot console.log() tests. Doing regular tests during development helps to trace the flow of the codes logic. The project came out very well.

**Comment by bessiewng** [comment]
@mp1pro Thanks for your comments Marvin. Is it good to be original and intuitive? Or are there best practices that I should have followed?

**Comment by mp1pro** [comment]
Yes, it is good to be original and intuitive. The code was great.
If you need more help, refer to react advisors at codecademy.com

Marvin Parke
Web Developer
cel phone: 3058963657
Github: https://github.com/mp1pro
Linkedin: https://www.linkedin.com/in/marvinparke
